src/Microsoft.HttpRepl.IntegrationTests/Commands/CommandHelper.cs

```diff
@@ -1,73 +1,29 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
 using System.Net.Http;
-using System.Threading;
-using System.Threading.Tasks;
 using Microsoft.HttpRepl.FileSystem;
 using Microsoft.HttpRepl.IntegrationTests.Mocks;
 using Microsoft.HttpRepl.IntegrationTests.Preferences;
 using Microsoft.HttpRepl.Preferences;
 using Microsoft.HttpRepl.UserProfile;
 using Microsoft.Repl;
-using Microsoft.Repl.Commanding;
 using Microsoft.Repl.ConsoleHandling;
 using Microsoft.Repl.Parsing;
 using Moq;
 
 namespace Microsoft.HttpRepl.IntegrationTests.Commands
 {
-    public class CommandHelper<T>
-        where T : ICommand<HttpState, ICoreParseResult>
+    public class CommandHelper
```

Comment:
This probably could have gone into the other PR, but I didn't want to delay it. It sort of feels weird to have a class named `*Helper` that is intended to be the base class for other classes. `*Helper` to me implies static methods or something to be used by something else. What do you think about either renaming it to something like `BaseCommandTests`/`CommandTestsBase` or making the three functions static (since I don't think they depend on anything anymore). cc @jimmylewis since I'm guessing he'll have opinions.